include/wx/generic/grideditors.h

```diff
@@ -55,7 +55,14 @@ class wxGridCellEditorEvtHandler : public wxEvtHandler
 class WXDLLIMPEXP_ADV wxGridCellTextEditor : public wxGridCellEditor
 {
 public:
-    explicit wxGridCellTextEditor(size_t maxChars = 0);
```

Comment:
Why were these explicit?

Reply:
Err, why not? All ctors taking a single argument should be `explicit` unless an implicit conversion is desirable and we *definitely* don't want to implicitly allow converting numbers to grid editors -- because this doesn't make any sense.

Reply:
I've always worked by "compiler knows best", but no problem - explicit is back